src/sentry/static/sentry/app/components/events/contexts/runtime/runtime.tsx

```diff
@@ -3,16 +3,29 @@ import React from 'react';
 import ContextBlock from 'app/components/events/contexts/contextBlock';
 
 import getRuntimeKnownData from './getRuntimeKnownData';
-import {RuntimeData, RuntimeKnownDataType} from './types';
+import {RuntimeData, RuntimeKnownDataType, RuntimeIgnoredDataType} from './types';
+import getUnknownData from '../getUnknownData';
 
 type Props = {
   data: RuntimeData;
 };
 
 const runTimerKnownDataValues = [RuntimeKnownDataType.NAME, RuntimeKnownDataType.VERSION];
 
-const Runtime = ({data}: Props) => (
-  <ContextBlock knownData={getRuntimeKnownData(data, runTimerKnownDataValues)} />
-);
+const runTimerIgnoredDataValues = [RuntimeIgnoredDataType.BUILD];
```

Comment:
Please isn't `runTimer` a typo?

Reply:
Probably yes, these variables were already called like this in the rest of the file - I'll change them all.